packages/shared/lib/core/layer-2/actions/pollLayer2NativeAssets.ts

```diff
@@ -0,0 +1,15 @@
+import { LAYER2_NATIVE_ASSETS_POLL_INTERVAL } from '../constants'
+import { getLayer2NativeTokens } from './'
+
+let pollInterval: number
+
+export function pollLayer2NativeAssets(): void {
```

Comment:
We should make a `pollLayer2Tokens` that will be responsible for getting both wrapped native tokens and native (to) L2 tokens.